lib/utils/src/graph/algorithms.cc

```diff
@@ -338,6 +338,18 @@ std::unordered_map<Node, std::unordered_set<Node>> get_dominators(IDiGraphView c
   return result;
 }
 
+std::unordered_set<Node> get_dominators(IDiGraphView const &, Node const &) {
```

Comment:
Let's not add this. I'd rather a linker error than silent failure

Reply:
Should I remove the other stubs as well? Do you want to only merge the makefile?

Reply:
> Should I remove the other stubs as well?

That or change them to `static_assert`s so we get compile time errors